internal/service/privatelinkendpointserviceserverless/data_source_privatelink_endpoints_service_serverless.go

```diff
@@ -25,16 +23,6 @@ func PluralDataSource() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-			"page_num": {
```

Comment:
Do we need to update the related datasource documentation?

Reply:
They are not in the documentation